_pages/business-units/18f/projects-partners/leading-projects.md

```diff
@@ -52,12 +52,9 @@ Fear not! We don’t expect you to do this alone. We have a lot of documentation
 Some things they’ll help you navigate, find, and use:
 
 * If you need documentation, look to the [handbook]({{site.baseurl}}) and our [guides](https://18f.gsa.gov/guides/).
-* If you need support in a specific skill set or help with deliverables, get help from the labs:
-    * [Writing lab](https://github.com/18F/writing-lab)
-    * [Design lab](https://github.com/18F/design-lab/)
-    * Dev lab ([#dev-lab](https://gsa-tts.slack.com/messages/dev-lab/) in Slack)
+* If you need support in a specific skill set or help with deliverables, get help by using a [microrequest](https://gsa-tts.slack.com/messages/microrequests/).
 * If your project is part of a portfolio, the portfolio director and your colleagues in the portfolio can be a great resource.
-* If you need advice or coaching, look to the chapters ([#product](https://gsa-tts.slack.com/messages/product/), [#design](https://gsa-tts.slack.com/messages/design/), [#acquisition](https://gsa-tts.slack.com/messages/acquisition/), [#dev](https://gsa-tts.slack.com/messages/dev/), and [#strategy](https://gsa-tts.slack.com/messages/strategy/)) and guilds ([#g-accessibility](https://gsa-tts.slack.com/messages/g-accessibility/), [#g-research](https://gsa-tts.slack.com/messages/g-research/), [#g-content](https://gsa-tts.slack.com/messages/g-content/), [#g-diversity](https://gsa-tts.slack.com/messages/g-diversity/), [#g-devops](https://gsa-tts.slack.com/messages/g-devops/).
+* If you need advice or coaching, look to the chapters ([#product](https://gsa-tts.slack.com/messages/product/), [#design](https://gsa-tts.slack.com/messages/design/), [#acquisition](https://gsa-tts.slack.com/messages/acquisition/), and [#dev](https://gsa-tts.slack.com/messages/dev/) and guilds ([#g-accessibility](https://gsa-tts.slack.com/messages/g-accessibility/), [#g-research](https://gsa-tts.slack.com/messages/g-research/), [#g-content](https://gsa-tts.slack.com/messages/g-content/), [#g-diversity](https://gsa-tts.slack.com/messages/g-diversity/), [#g-devops](https://gsa-tts.slack.com/messages/g-devops/).
```

Comment:
We're missing eng-practices and security & compliance. But I wonder if it'd be better to just link over to the guilds and working groups page, so we don't have to worry 'bout keeping this list up to date.

Reply:
Fair point, though not sure whether it makes more sense to link to the list of chapters or working groups and guilds. Feel free to send a pull request if you feel strongly one way or another.

Reply:
My point is that this is an incomplete list. It would be better to publish without a list and just reference the canonical source. I'll open a follow-up PR.